### ISSUE TITLE
Developer account issue

### ISSUE THREAD

**Issue by Daniel-0011** [rileytestut/AltStore#1044]
I just signed with a developer account to Altstore, not a free developer account. but in Altstore settings it shows a free developer account. that's why I cannot inject more than 3 apps and have to refresh the apps within 7 days.

**Comment by lonkelle** [comment]
Do you have an Organization account?

**Comment by shanegillio** [comment]
this is an ongoing repeat issue noted by some with dev accounts linked to an org. going to close this and will have pinned some info on this issue later so stay tuned. hopefully we can find a permanent resolution soon

**Comment by lonkelle** [comment]
Of note, this is a 3 - 4 line code fix that @osy already PRed nearly a year ago (https://github.com/rileytestut/AltStore/pull/967). Since it's still open, all that Riley has to do, is hit "Merge Changes" - I've personally tested osy's code and it works as intended.

**Comment by shanegillio** [comment]
thanks - will review this then and see about implementing

**Comment by lonkelle** [comment]
**NOTE:** Osy's PR only picks the first in the list of organizations while many Apple ID's are connected to multiple. When Riley talked about adding this feature in 2021 he specifically wanted to make a UI "Team Chooser" so you'll have to run that by him again to refresh his memory.

**Comment by osy** [comment]
I agree an account picker is the better solution but I think the current situation is that free account is preferred over organization account. If I remember correctly from conversations on discord, there was some concerns about certificates and identifiers and not wanting AltStore to accidentally delete something from an org account.

FWIW I’ve been running a patched AltStore with my PR applied and haven’t had any issues.

**Comment by shanegillio** [comment]
good to know - definitely going to prioritize getting this once I can confirm with Riley

**Comment by itrack** [comment]
Hi,
Any updates?

Thank you!

**Comment by fylo5** [comment]
Thx

**Comment by osy** [comment]
FWIW, the profile you choose will be saved so the workaround I've been using is:

1. Build my own AltStore with the PR
2. Sign it with the same provisioning profile and bundle id as downloaded AltStore (make sure to change Info.plist to point to your group container id and device id and altserver id manually)
3. Install it on device, log in, then install released AltStore. As long as you don't delete it, you can continue to get AltStore updates without issues.

Note the first time you install it you'll get double AltStore because of the different account used to sign it. Just delete the first one.

**Comment by matejzvikl23** [comment]
mmm

**Comment by omfatimh38** [comment]
> I just signed with a developer account to Altstore, not a free developer account. but in Altstore settings it shows a free developer account. that's why I cannot inject more than 3 apps and have to refresh the apps within 7 days.

**Comment by zviklmatej07-dev** [comment]
icif
